indra/resources/default_belief_probs.json

```diff
@@ -65,6 +66,7 @@
     "omnipath": 0.1,
     "biofactoid": 0.1,
     "minerva": 0.1,
-    "dgi": 0.1
+    "dgi": 0.1,
+    "crog": 0.01
```

Comment:
maybe i'm tooting my own horn, but I'm very confident in the curation I did here